script.js

```diff
@@ -0,0 +1,578 @@
+        KeyR: 'к',
+        KeyT: 'е',
+        KeyY: 'н',
+        KeyU: 'г',
+        KeyI: 'ш',
+        KeyO: 'щ',
+        KeyP: 'з',
+        BracketLeft: 'х',
+        BracketRight: 'ъ',
+        Backslash: '\\',
+        Delete: 'del',
+        CapsLock: 'CapsLock',
+        KeyA: 'ф',
+        KeyS: 'ы',
+        KeyD: 'в',
+        KeyF: 'а',
+        KeyG: 'п',
+        KeyH: 'р',
+        KeyJ: 'о',
+        KeyK: 'л',
+        KeyL: 'д',
+        Semicolon: 'ж',
+        Quote: 'э',
+        Enter: 'Enter',
+        ShiftLeft: 'Shift',
+        KeyZ: 'я',
+        KeyX: 'ч',
+        KeyC: 'с',
+        KeyV: 'м',
+        KeyB: 'и',
+        KeyN: 'т',
+        KeyM: 'ь',
+        Comma: 'б',
+        Period: 'ю',
+        Slash: '.',
+        ShiftRight: 'Shift',
+        ControlLeft: 'Ctrl',
+        win: 'win',
+        AltLeft: 'Alt',
+        Space: ' ',
+        AltRight: 'Alt',
+        ControlRight: 'Ctrl',
+        ArrowUp: '&#8593;',
+        ArrowRight: '&#8594;',
+        ArrowDown: '&#8595;',
+        ArrowLeft: '&#8592;',
+      },
+      shift: {
+        Backquote: 'Ё',
+        Digit1: '!',
+        Digit2: '"',
+        Digit3: '№',
+        Digit4: ';',
+        Digit5: '%',
+        Digit6: ':',
+        Digit7: '?',
+        Digit8: '*',
+        Digit9: '(',
+        Digit0: ')',
+        Minus: '_',
+        Equal: '+',
+        BackSpace: 'BackSpace',
+        Tab: 'Tab',
+        KeyQ: 'Й',
+        KeyW: 'Ц',
+        KeyE: 'У',
+        KeyR: 'К',
+        KeyT: 'Е',
+        KeyY: 'Н',
+        KeyU: 'Г',
+        KeyI: 'Ш',
+        KeyO: 'Щ',
+        KeyP: 'З',
+        BracketLeft: 'Х',
+        BracketRight: 'Ъ',
+        Backslash: '/',
+        Delete: 'del',
+        CapsLock: 'CapsLock',
+        KeyA: 'Ф',
+        KeyS: 'Ы',
+        KeyD: 'В',
+        KeyF: 'А',
+        KeyG: 'П',
+        KeyH: 'Р',
+        KeyJ: 'О',
+        KeyK: 'Л',
+        KeyL: 'Д',
+        Semicolon: 'Ж',
+        Quote: 'Э',
+        Enter: 'Enter',
+        ShiftLeft: 'Shift',
+        KeyZ: 'Я',
+        KeyX: 'Ч',
+        KeyC: 'С',
+        KeyV: 'М',
+        KeyB: 'И',
+        KeyN: 'Т',
+        KeyM: 'Ь',
+        Comma: 'Б',
+        Period: 'Ю',
+        Slash: ',',
+        ShiftRight: 'Shift',
+        ControlLeft: 'Ctrl',
+        win: 'win',
+        AltLeft: 'Alt',
+        Space: ' ',
+        AltRight: 'Alt',
+        ControlRight: 'Ctrl',
+        ArrowUp: '&#8593;',
+        ArrowRight: '&#8594;',
+        ArrowDown: '&#8595;',
+        ArrowLeft: '&#8592;',
+      },
+    },
+  },
+
+
+  // method for creating new elements
+  createNewElement(tag, tagClass, tagText) {
+    const elm = document.createElement(tag);
+    elm.className = tagClass;
+    elm.innerHTML = tagText;
+    return elm;
+  },
+
+
+  // method for create KexBox element, arguments:array keys, array function keys
+  createKeyBox(arrKeys, arrFuncKeys) {
+    const keyBox = this.createNewElement('div', 'keyboard', '');
```

Comment:
I think named argements are better choise in this case:
```
createNewElement({ tag, tagClass, tagText }){...}
 const keyBox = this.createNewElement({ tag:'div', tagClass: 'keyboard', tagText: '' });
```

Reply:
Done

Reply:
:thumbsup: